content/en/tracing/trace_collection/otel_instrumentation/ruby.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Custom Instrumentation of Ruby Applications with the OpenTelemetry API
+kind: documentation
+description: 'Instrument your Ruby application with OTel API to send traces to Datadog.'
+code_lang: ruby
+type: multi-code-lang
+code_lang_weight: 20
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
+
+{{% otel-custom-instrumentation %}}
+
+
+## Requirements and limitations
+
+- Datadog Ruby tracing library `dd-trace-rb` version 1.9.0 or greater
+- Gem version support 1.1.0 or greater
```

Comment:
```suggestion
- Datadog Ruby tracing library `dd-trace-rb` version 1.9.0 or greater.
- Gem version support 1.1.0 or greater.
```

Reply:
The suggestions for periods is because I saw that you used them in the Java article. But the others don't use periods, and I could personally go either way.